resolwe_bio/processes/variant_calling/vc_chemut.yml

```diff
@@ -282,6 +285,7 @@
       java \
         -jar /usr/gitc/GATK36.jar \
         -T UnifiedGenotyper ${SAMPLES} \
+        -nct {{ requirements.resources.cores }} \
```

Comment:
Remove the `-nct` option from `UnifiedGenotyper`, since issues with the parallelization usage have been reported for this tool. 

_Many users have reported issues running HaplotypeCaller with the -nct argument, so we recommend using Queue to parallelize HaplotypeCaller instead of multithreading._